README.md

```diff
@@ -27,7 +27,8 @@ mkdir -p catkin_ws/src
 
 # Clone repository into source folder
 cd catkin_ws/src
-git clone git@github.com:UHHRobotics22-23/robot_project 
+git clone git@github.com:UHHRobotics22-23/robot_project
+cd ..
```

Comment:
is this correct on linux?
I am used to cd robot_project

Reply:
We don't want to go inside the repo, but back to the workspace. In Linux `..` goes one directory up.